### PR TITLE
Adds a switch element for managing physics 🌎

### PR DESCRIPTION
**Pull request by margaritiko**
Adds a switch element for managing physics.

**Main changes:**
* Utilised custom switch element from Bootstrap and extended `index.css` file with switch styles.
* Saved vis.js options in a separate constant field.
* Added event listener for switch.

switch

### CODE REVIEW

**Review by kwchan1995 — APPROVED** [review]

File: project/src/com/google/daggerqueryui/scripts/binding_graph.js [hunk 1]
```diff
@@ -118,9 +123,29 @@ const bindingGraph = (function() {
...
+  /**
+   * Checks if an edge exists in the graph or not.
+   *
+   * @param {number} sourceId
+   * @param {number} targetId
+   * @return {boolean}
+   */
+  function isEdgeExists(sourceId, targetId) {
+    return edges.get({
+      filter: function (edge) {
```
Comment (o-y) [inline project/src/com/google/daggerqueryui/scripts/binding_graph.js]: Could be simplified into a lambda using the js [arrow notation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) if you want

File: project/src/com/google/daggerqueryui/scripts/binding_graph.js [hunk 2]
```diff
@@ -118,9 +123,29 @@ const bindingGraph = (function() {
...
   }
 
+  /**
+   * Checks if an edge exists in the graph or not.
+   *
+   * @param {number} sourceId
+   * @param {number} targetId
+   * @return {boolean}
+   */
+  function isEdgeExists(sourceId, targetId) {
```
Comment (Ratchette) [inline project/src/com/google/daggerqueryui/scripts/binding_graph.js]: nit: can we rename this to edgeExists? or hasEdge?

**Review by Ratchette — APPROVED** [review]

File: project/src/com/google/daggerqueryui/scripts/binding_graph.js (hunk 1 above)
Comment (margaritiko) [reply project/src/com/google/daggerqueryui/scripts/binding_graph.js]: Definitely it looks better, thanks! [Done](https://github.com/googleinterns/dagger-query/pull/45/commits/ef15c524f5c8194d5ddb68958b11bed1e01642fe)

File: project/src/com/google/daggerqueryui/scripts/binding_graph.js (hunk 2 above)
Comment (margaritiko) [reply project/src/com/google/daggerqueryui/scripts/binding_graph.js]: Oops, I see what's wrong, thank you!  
[Done](https://github.com/googleinterns/dagger-query/pull/45/commits/fc25fd9b4fdd53339f3fdfdf1d4e812ddef3c2dd) 💎

**Pull request merged**